src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+public class Node<K, V> {
+    private final K key;
+    private V value;
+    private Node<K, V> next;
+
+    public Node(K key, V value, Node<K, V> next) {
+        this.key = key;
+        this.value = value;
+        this.next = next;
+    }
+
+    public K getKey() {
+        return key;
+    }
+
+    public V getValue() {
+        return value;
+    }
+
+    public void setValue(V value) {
+        this.value = value;
+    }
+
+    public Node<K, V> getNext() {
+        return next;
+    }
+
+    public void setNext(Node<K, V> next) {
+        this.next = next;
+    }
```

Comment:
Don't use getters and setters in class Node. We have access to private fields of the inner class, so using getters and setters is redundant.